jujugui/static/gui/src/app/views/inspectors/service-inspector.js

```diff
@@ -98,8 +100,10 @@ YUI.add('service-inspector', function(Y) {
         this.views.overview.render(model, this.getAttrs());
       }
       // Hide the 'Change version' button if the service is pending.
-      this.get('container').one('.change-version-trigger').toggleClass(
-          'hidden', model.get('pending'));
+      container.one('.change-version-trigger').toggleClass('hidden', pending);
+      // Set the 'Destroy service' button to full width if there is no
+      // 'Change version' button.
+      container.one('.destroy-service-trigger').toggleClass('wide', pending);
```

Comment:
Can't this behaviour created with flexbox? By setting the buttons to flex auto they will take up the space available.